ParisBNF/abb/BNFabb92.xml

```diff
@@ -0,0 +1,682 @@
+                            <incipit xml:lang="gez">
+                                <locus from="1ra"/><hi rend="rubric">አመ፡ ሠሉሰ፡ ለኅዳር፡</hi> አስተርእዮቱ፡
+                                    ለአን<hi rend="rubric">ባቆም፡ ነቢይ፡ በአ</hi>ማን፡ በሃገረ፡ እንተ፡ ስ<hi
+                                    rend="rubric">ማ፡ ቀርጠሳ፡ እንዘ፡ ሊቀ፡ ጳጳሳት፡ አባ፡ አርሚስ፡</hi>
+                            </incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus target="#16rb"/>
+                            <title type="complete" xml:lang="en" ref="LIT6170GadlaEleazar">
+                                Commemoration of <persName ref="PRS13141Eleazar">Eleazar</persName>,
+                                the son of Aaron (<date ref="ethiocal:Hedar3">3rd of
+                                Ḫǝdār</date>)</title>
+                            <incipit xml:lang="gez">ወበይእቲ፡ ዕለት፡ ገድለ፡ አልዐዛር፡ ወልደ፡ አሮን፡ ወተረክበ፡ ነገሮ፡
+                                በውስተ፡ ኦሪት፡ ስብሓት፡ ለእግዚአብሔር፡ ለዓለመ፡ ዓለም፡ አሜን።</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="16va" to="36va"/>
+                            <title xml:lang="en" type="complete" ref="LIT2487Vision">Discourse of
+                                    <persName ref="PRS9486Theophil">Theophilus of
+                                    Alexandria</persName>, on the blessed Virgin Mary (<date
+                                    ref="ethiocal:Hedar6">6h of Ḫǝdār</date>)</title>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">አመ፡ ፯፡ ለኅዳር፡ ምንበብ፡ በስመ፡ አብ፡
+                                    ወወልድ፡ ወመ</hi>ንፈስ፡ ቅዱስ፡ አሓዱ፡ አ<hi rend="rubric">ምላክ፡ ንቀድም፡
+                                    በረድ</hi>ኤተ፡ እግዚአብሔር፡ ኢየሱ<hi rend="rubric">ስ፡ ክርስቶስ፡ ንጽሐፍ፡
+                                </hi>ድርሳን፡ ዘቅዱስ፡ ወብፁ<hi rend="rubric">ዕ፡ ቴኦፍሎስ፡ ሊቀ፡ ጳ</hi>ጳሳት፡ ዘሀገር፡
+                                እለ፡ ስክንድርያ፡ ጸሎቱ፡ ትብጽሐነ፡ ለዓለመ፡ ዓለም፡ አሜን። ዘይቤ፡ በእንተ፡ እግዝእትነ፡ ቅድስት፡
+                                ድንግል፡ ማርያም፡ ወበእንተ፡ ቤት፡ ዘኀደረት፡ ምስለ፡ ፍቁር፡ ወልዳ፡ ኢየሱስ፡ ክርስቶስ፡ በደብረ፡
+                                ቊስቋም።</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="36vb" to="57vb"/>
+                            <title type="complete" xml:lang="en" ref="LIT7298EncomiumTammeraG">The
+                                encomium on St. George of Lydda by Bishop Theodosius, followed by 12
+                                (canonical) miracles</title>
+                            <msItem xml:id="ms_i4.1">
+                                <locus from="36vb" to="39ra"/>
+                                <title type="complete" xml:lang="en" ref="LIT6210MGTranslation">The
+                                    encomiumon St. George of Lydda by Bishop Theodosius </title>
```

Comment:
encomium :)
Did you carry out the spelling check? :)

Reply:
ah! no, I forgot this function exist!